src/activities/binary_bulb/LightBulb.qml

```diff
@@ -0,0 +1,82 @@
+/* GCompris - LightBulb.qml
+ *
+ * Copyright (C) 2018 Rajat Asthana <rajatasthana4@gmail.com>
+ *
+ * Authors:
+ *   RAJAT ASTHANA <rajatasthana4@gmail.com>
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.6
+import "../../core"
+import GCompris 1.0
+
+Image {
+    id: bulb
+    anchors.verticalCenter: parent.verticalCenter
+    sourceSize.height: 60 * ApplicationInfo.ratio
+    source: "resource/bulb_off.svg"
+    state: "off"
+
+    property string bit: ""
+    property int value: Math.pow(2,items.numberOfBulbs-index-1)
+
+    GCText {
+        anchors.bottom: parent.top
+        anchors.horizontalCenter: parent.horizontalCenter
+        text: value
+        color: "white"
+        visible: items.score.currentSubLevel <= 2 ? true : false
```

Comment:
this text should always be visible.
We probably don't expect the child to know the power of two values